INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:24] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:25] "GET /static/css/custom.css HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:25] "GET /static/css/theme.css HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:25] "GET /static/js/custom.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:25] "GET /static/js/new_ajax_helper.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:26] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:flask_app:[{'userid': 1, 'name': 'admin', 'email': 'admin@admin', 'password': 'AVerySecurePassword', 'permission': 'admin'}]
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:38] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:38] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:38] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:38] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:38] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "[32mGET /dashboard HTTP/1.1[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:40] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:flask_app:[{'userid': 1, 'name': 'admin', 'email': 'admin@admin', 'password': 'AVerySecurePassword', 'permission': 'admin'}]
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:47] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:47] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:47] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:47] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:00:47] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:flask_app:[{'userid': 1, 'name': 'admin', 'email': 'admin@admin', 'password': 'AVerySecurePassword', 'permission': 'admin'}]
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "[32mPOST / HTTP/1.1[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "GET /dashboard HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "GET /static/js/dashboard.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "GET /static/js/keybind.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "GET /static/js/mapping.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:35] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:flask_app:LOADING CAMERA
INFO:flask_app:FLASK APP: CAMERA NOT WORKING
INFO:flask_app:FLASK APP: LOADING THE ROBOT
INFO:flask_app:FLASK APP: LOADING THE SOUND
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:38] "[35m[1mPOST /robotload HTTP/1.1[0m" 500 -
ERROR:werkzeug:Error on request:
Traceback (most recent call last):
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 324, in run_wsgi
    execute(self.server.app)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 313, in execute
    application_iter = app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\matth\OneDrive\Desktop\Code\Robot\flask_app.py", line 62, in robotload
    GLOBALS.SOUND = soundinterface.SoundInterface()
NameError: name 'soundinterface' is not defined
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:43] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:01:43] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "GET /dashboard HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/js/dashboard.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "[36mGET /static/js/mapping.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:11] "GET /static/js/keybind.js HTTP/1.1" 200 -
INFO:flask_app:LOADING CAMERA
INFO:flask_app:FLASK APP: CAMERA NOT WORKING
INFO:flask_app:FLASK APP: LOADING THE ROBOT
INFO:flask_app:FLASK APP: LOADING THE SOUND
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:02:13] "[35m[1mPOST /robotload HTTP/1.1[0m" 500 -
ERROR:werkzeug:Error on request:
Traceback (most recent call last):
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 324, in run_wsgi
    execute(self.server.app)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 313, in execute
    application_iter = app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\matth\OneDrive\Desktop\Code\Robot\flask_app.py", line 62, in robotload
    GLOBALS.SOUND = soundinterface.SoundInterface()
NameError: name 'soundinterface' is not defined
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:04] "GET /dashboard HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/js/dashboard.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "[36mGET /static/js/mapping.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:05] "GET /static/js/keybind.js HTTP/1.1" 200 -
INFO:flask_app:LOADING CAMERA
INFO:flask_app:FLASK APP: CAMERA NOT WORKING
INFO:flask_app:FLASK APP: LOADING THE ROBOT
INFO:flask_app:FLASK APP: LOADING THE SOUND
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:03:06] "[35m[1mPOST /robotload HTTP/1.1[0m" 500 -
ERROR:werkzeug:Error on request:
Traceback (most recent call last):
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 324, in run_wsgi
    execute(self.server.app)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 313, in execute
    application_iter = app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\matth\OneDrive\Desktop\Code\Robot\flask_app.py", line 62, in robotload
    GLOBALS.SOUND = soundinterface.SoundInterface()
NameError: name 'soundinterface' is not defined
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "GET /dashboard HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/css/theme.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/css/custom.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/js/new_ajax_helper.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/js/custom.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/js/dashboard.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "[36mGET /static/js/mapping.js HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:52] "GET /static/js/keybind.js HTTP/1.1" 200 -
INFO:flask_app:LOADING CAMERA
INFO:flask_app:FLASK APP: CAMERA NOT WORKING
INFO:flask_app:FLASK APP: LOADING THE ROBOT
INFO:flask_app:FLASK APP: LOADING THE SOUND
INFO:werkzeug:127.0.0.1 - - [08/Mar/2022 15:04:53] "[35m[1mPOST /robotload HTTP/1.1[0m" 500 -
ERROR:werkzeug:Error on request:
Traceback (most recent call last):
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 324, in run_wsgi
    execute(self.server.app)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\werkzeug\serving.py", line 313, in execute
    application_iter = app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\matth\AppData\Local\Programs\Python\Python39\Lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\matth\OneDrive\Desktop\Code\Robot\flask_app.py", line 62, in robotload
    GLOBALS.SOUND = soundinterface.SoundInterface()
NameError: name 'soundinterface' is not defined
INFO:werkzeug: * Detected change in 'C:\\Users\\matth\\OneDrive\\Desktop\\Code\\Robot\\flask_app.py', reloading
INFO:werkzeug: * Restarting with stat
